### ISSUE TITLE
Investigate possibility of copying over vanilla loadout into Northstar for new users

### ISSUE THREAD

**Issue by GeckoEidechse** [R2Northstar/Northstar#470]
For new Northstar users it could heavily reduce the friction if we could somehow copy their pdata (loadouts in particular) over to Northstar on first launch.

Cause as such they wouldn't have to set up their loadouts again and could go straight into playing the game.

Optimally this would be done on masterserver. Masterserver already sees when a player is new to Northstar as there's no database entry for said user on it.

Additionally to ensure Titanfall2 ownership, client also sends Stryder token(?) to masterserver for verification. If that token has enough perms, it could be enough to request player loadout from Stryder, copy pdata into masterserver db, and send that pdata to client the same way it would if pdata was stored on masterserver to begin with.

Additionally it would be nice if clients had an option to sync their pdata from vanilla to Northstar at a later moment though this is lower priority and mostly just requires adding UI in client and API endpoint on masterserver for it.

**Comment by GeckoEidechse** [comment]
One risk to consider with this is that depending on how many new players we get within a certain timeframe we could actually get ratelimited/blocked by Stryder.

**Comment by pg9182** [comment]
This should be done client-side IMO.

- No concern about rate limits / blocking.
- Player remains in control over player data.
- Possibly easier to implement.
- Cleaner overall.
